### PR TITLE
Docs for Basal Ganglia Population Morphology Validation Use Case

### PR DESCRIPTION
**Pull request by pedroernesto**

### CODE REVIEW

**Comment by lbologna** [comment]
@pedroernesto @alex4200 I just accepted this changes and push it to DEV:
https://lbologna.github.io/hbp-sp6-guidebook/online_usecases/validation_framework/validation_framework.html

Please let me know if it is fine with you before moving to PROD.
Thanks

**Comment by alex4200** [comment]
It looks like there are four small typos in the new documentation: 

- Twice 'dimensionl' instead of 'dimensional'
- Twice 'indiviual' instead of 'individual'

File: source/online_usecases/validation_framework/validation_framework.rst [hunk 1]
```diff
@@ -184,7 +184,26 @@ We currently have the following Use Cases on the Collaboratory:
 
     Some of the features currently included are soma's diameter and the maximal branch order in the dendrites, besides the number of trunk sections, -X,Y,Z- extents, field's diameter and total path-length of both the axon and the dendrites.
 
-    **Note:** Currently only FS interneurons can be considered, since observation data is missing for other neuron types.
+    **Note:** Currently only Striatum Fast-Spiking Interneurons (FSI) can be considered, since observation data is missing for other neuron types.
+
+**6) Basal Ganglia Population Morphology Validation**
+  The average morphometrics of a population of (digitally reconstructed) Fast-Spiking Interneurons (FSI) in Striatum, is validated against experimental data. Additional plots are provided to visualize some statistics derived from the morphometrics of the indiviual cells, e.g. linear regression analysis, histograms and Kernel-Distribution-Estimates (KDE) for single features, and bi-dimensionl joint KDEs for pairs of uncorrelated features.
```
Comment (alex4200) [inline source/online_usecases/validation_framework/validation_framework.rst]: indiviual cells -> individual cells
dimensionl -> dimensional ?

File: source/online_usecases/validation_framework/validation_framework.rst [hunk 2]
```diff
@@ -184,7 +184,26 @@ We currently have the following Use Cases on the Collaboratory:
...
+
+  - **Soft Constraints** *[Currently only available for Fast Spiking Interneurons]*
+    Neuronal reconstructions that pass the 'Hard Constraints' are evaluated here for their morphometric features. The features are extracted using NeuroM's morph_stats app, wherever possible, either directly or as a combination of multiple features. The average morphometrics of the population of neurons is then computed. Those mean values are then compared against experimentally obtained data, as determined by the particular validation test being executed.
+
+    Some of the features currently included are soma's diameter and the maximal branch order in the dendrites, besides the number of trunk sections, -X,Y,Z- extents, field's diameter and total path-length of both the axon and the dendrites.
+
+    Additional plots are provided to visualize some statistics derived from the morphometrics of the indiviual cells, e.g. linear regression analysis, histograms and Kernel-Distribution-Estimates (KDE) for single features, and bi-dimensionl joint KDEs for pairs of uncorrelated features.
```
Comment (alex4200) [inline source/online_usecases/validation_framework/validation_framework.rst]: indiviual cells -> individual cells
dimensionl -> dimensional ?

Comment (pedroernesto) [reply source/online_usecases/validation_framework/validation_framework.rst]: It is corrected, thanks

**Pull request merged**